pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -261,6 +261,9 @@ with generated;
       asmfmt
       delve
       errcheck
+      go-tools
+      gocode
+      gocode-gomod
```

Comment:
What does gocode-gomode?

Reply:
I was taken by this as well. gocode-gomod is a fork of gocode with support for Go modules. Unfortunately, neither project can replace the other!

See gocode README https://github.com/mdempsky/gocode#githubcommdempskygocode

Reply:
Is vim-go smart enough to switch between the too?

Reply:
yes, it is. it installs it [here](
https://github.com/fatih/vim-go/blob/8513febab5c698553acc7ac33f4185fb2da664d2/plugin/go.vim#L50) and uses it [](
https://github.com/fatih/vim-go/blob/ab0eb131ae5f3e2b7c069291488d6636cc717843/autoload/go/complete.vim#L14).